meson.build

```diff
@@ -1712,6 +1713,12 @@ exe = executable('systemctl', 'src/systemctl/systemctl.c',
                  install_dir : rootbindir)
 public_programs += [exe]
 
+foreach alias : ['halt', 'init', 'poweroff', 'reboot', 'runlevel', 'shutdown', 'telinit']
+        meson.add_install_script(meson_make_symlink,
+                                 join_paths(rootbindir, 'systemctl'),
```

Comment:
Note. Currently, `init` is a symlink to `/usr/lib/systemd/systemd` on Fedora, not `systemctl`.

Reply:
Oh, oh, oh. Thanks for catching this. Of course.